app/models/system-notification.js

```diff
@@ -0,0 +1,18 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+export default class SystemNotificationModel extends Model {
+  @attr('string') uri;
+
+  @attr('string') subject;
+  @attr('string') message;
+
+  @attr('datetime') createdAt;
+  @attr('datetime') readAt;
+  @attr('datetime') archivedAt;
+
+  @belongsTo('gebruiker', {
+    async: true,
+    inverse: 'system-notification',
```

Comment:
I think this should be:
`inverse: 'systemNotifications',` 
no?

Reply:
If you look at other models indeed this should be `systemNotifications` instead of `system-notifications. Changed, it looks like it didn't make a difference but now its in line with the rest :D